src/core/execute.h

```diff
@@ -265,7 +279,7 @@ struct ExecParameters {
         CGroupMask cgroup_supported;
         const char *cgroup_path;
 
-        const char *runtime_prefix;
+        char *prefix[_EXEC_DIRECTORY_MAX];
```

Comment:
if you define this as `char**` then we can assign a pointer to the Manager's prefix array here, and don#t have to copy the array at all!